src/dmd/mars.d

```diff
@@ -2379,3 +2408,43 @@ Modules createModules(ref Strings files, ref Strings libmodules)
     }
     return modules;
 }
+
+bool parseCLIOption(string name, Usage.Feature[] features)(ref Param params, const(char)* p)
+{
+    // Parse:
+    //      -<name>=<feature>
+    const ps = p + name.length + 1;
+    if (*ps != '=')
+        return false;
+
+    if (strcmp(ps + 1, "?") == 0)
```

Comment:
This can use `isHelpOption` once https://github.com/dlang/dmd/pull/9205 has been merged.